Gems/AWSCore/Code/Platform/Android/GetJsonSchemaPath_Android.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/IO/FileIO.h>
+#include <AzCore/IO/Path/Path.h>
+#include <AzCore/IO/SystemFile.h>
+
+namespace AWSCore
+{
+    namespace Platform
+    {
+        AZ::IO::Path GetJsonSchemaPath()
+        {
+            static constexpr const char ResourceMapppingJsonSchemaFilePath[] =
+                "@products@/resource_mapping_schema.json";
+
+            AZ::IO::FixedMaxPath resolvedSchemaPath;
+            if (!AZ::IO::FileIOBase::GetDirectInstance()->ResolvePath(resolvedSchemaPath, ResourceMapppingJsonSchemaFilePath))
+            {
+                return "";
```

Comment:
This can just return `AZ::IO::Path{}` which is slightly more efficient as it a string length doesn't have to be performed as it would with an empty string..
```suggestion
                return AZ::IO::Path;
```

Reply:
Fixed.